governance/program/tests/program_test/tools.rs

```diff
@@ -35,3 +35,11 @@ pub fn map_transaction_error(transport_error: TransportError) -> ProgramError {
         _ => panic!("TEST-TRANSPORT-ERROR: {:?}", transport_error),
     }
 }
+
+pub fn clone_keypair(source: &Keypair) -> Keypair {
+    Keypair::from_base58_string(&source.to_base58_string())
```

Comment:
nit: can you use `from_bytes(source.to_bytes())` instead?  Otherwise, I'm surprised that this doesn't just support clone...

Reply:
Changed. 
I've been wondering about the same myself.  Is cloning Keypair outside of tests kind of controversial? and it's better to keep the private key in one place?

Reply:
Yeah, that's a good point. It's then just the convenience of Cloning what  is missing.